plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeWriter.java

```diff
@@ -169,8 +169,8 @@ public long getRowCount()
     public DataFileInfo getDataFileInfo()
             throws IOException
     {
-        List<String> dataColumnNames = columnHandles.stream().map(DeltaLakeColumnHandle::getName).collect(toImmutableList());
-        List<Type> dataColumnTypes = columnHandles.stream().map(DeltaLakeColumnHandle::getType).collect(toImmutableList());
+        List<String> dataColumnNames = columnHandles.stream().map(DeltaLakeColumnHandle::getPhysicalName).collect(toImmutableList());
```

Comment:
Can you explain why you did this change? `getName` -> `getPhysicalName` & `getType` -> `getPhysicalType` .
Can/should this change be extracted to a separate PR?

cc @ebyhr

Reply:
Parquet writer uses physical name and type to comply [Respect Column Mapping](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#column-mapping) with `delta.columnMapping.mode=id` or `name` When `delta.columnMapping.mode` is none the methods provide logical values of name and type. That allows to use the methods in all modes of data writening: since when `delta.columnMapping.mode=id` or `name` it provides physical values, and `delta.columnMapping.mode=none` logical. 
This change should not be separated due directly is related to writer5 support of that PR.